openssh/files/sshd_config

```diff
@@ -128,8 +128,14 @@
 {{ option_default_uncommented('RSAAuthentication', 'yes') }}
 {{ option_default_uncommented('PubkeyAuthentication', 'yes') }}
 {{ option('AuthorizedKeysFile', '%h/.ssh/authorized_keys') }}
-{{ option('AuthorizedKeysCommand', 'none') }}
-{{ option('AuthorizedKeysCommandUser', 'nobody') }}
+{%- if server.authorized_keys_command is defined %}
+AuthorizedKeysCommand {{ server.authorized_keys_command.command }}
+{% if grains['os_family'] == 'RedHat' %}
```

Comment:
This deactivates AuthorizedKeysCommandUser for all distros except RedHat.

Why not

```jinja
if grains['os_family'] != 'RedHat' or grains['osmajorrelease'] != '6'
```
instead of two `if`s?